Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM docker.io/node:22.14.0-alpine
```

Comment:
You could probably template this to pick up the node version from `.node-version` file.

Reply:
I _think_ you can do something like this:
```
ARG NODE_VERSION
FROM node:${NODE_VERSION}-alpine
```

Then when you call the docker build, you'd pass in the value
```
docker build --build-arg NODE_VERSION=$(cat .node-version) -t blah .
```

Not sure how to do that in GitHub actions though.

Reply:
I’ll look into that tonight. Started learning more about GitHub actions last night, so have a good starting point!

Reply:
Got it! Took a few tries to get things just right, but the build is working, pulling node version from the included file, with a backup of the "lts" tag, which currently maps to 22.14.0. The fallback might eventually point to a different version, but it shouldn't matter since pulling the value from the file is working.